pkg/nuctl/command/common/importreport.go

```diff
@@ -0,0 +1,137 @@
+/*
+Copyright 2023 The Nuclio Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package common
+
+import (
+	"context"
+	"fmt"
+	"sync"
+
+	"github.com/nuclio/logger"
+)
+
+type ProjectReports struct {
+	Report
+	Reports map[string]*ProjectReport
+}
+
+func NewProjectReports() *ProjectReports {
+	return &ProjectReports{
+		Reports: make(map[string]*ProjectReport),
+	}
+}
+
+func (psr *ProjectReports) AddReport(report *ProjectReport) {
```

Comment:
```suggestion
func (pr *ProjectReports) AddReport(report *ProjectReport) {
```

Reply:
@TomerShor already used for `(pr *ProjectReport)`

Reply:
I don't think it really matters as you are specifying the pointer type explicitly. It shouldn't cause any conflicts

Reply:
okay, renamed then, same for functions